### ISSUE TITLE
Error: write EPIPE

### ISSUE THREAD

**Issue by boldt** [nodemailer/nodemailer#977]
Currently I am getting the following error with `sendmail: true` in my local ubuntu as well as in a docker container.

```
Error { Error: write EPIPE
    at WriteWrap.afterWrite [as oncomplete] (net.js:832:14) errno: 'EPIPE', code: 'EPIPE', syscall: 'write' }
```

Here you find a Docker-based MWE, so you can reproduce the error:

* https://github.com/boldt/nodemailer-issue-977

**Comment by andris9** [comment]
This means that writing to the stdin of sendmail binary does not work. Sendmail support is as is. If it does not work then there is something wrong with the system config. There is nothing to do on Nodemailer side - Nodemailer just tries to open the *stdin* of sendmail binary and write to it. If it fails then it fails.

**Comment by wallacepreston** [comment]
@boldt did you figure out what your issue was?  I'm getting the same error.  Only running in Docker, not locally in Mac OS.  It seemed to start when I upgraded to Node 12.

**Comment by boldt** [comment]
@wallacepreston I switched to the SMTP transport since I did not solve it.

**Comment by wallacepreston** [comment]
@boldt thanks for getting back to me!  I discovered my issue was not with the nodemailer package at all.  I was trying to send a PDF, and the PDF creation was failing.  That was what caused the error.  Unfortunately the html-pdf package hasn't been updated, so in Node 12 it failed.  I switched to Puppeteer for the PDF creation, and even then I had to do explicit installations in the Docker container running Linux Node-Alpine.
